File: coding-test/nodejs/src/practice/NSpaceByX.js
function solution(x, n){
    let answer = [];
    let num = x;
    
    for(let i = 0; i < n; i++){
        answer.push(num);
        num = num + x;
    }

    return answer;
}

function differentSolution(x, n){
    return Array(n).fill(x).map((v, i) => (i + 1) * v)
}

console.log(solution(2, 5))

File: practice-coding-test/java-kotlin/src/kr/co/programmers/java/practice/NSpaceByX.java
package kr.co.programmers.java.practice;

import java.util.Arrays;

public class NSpaceByX {

    public static void main(String[] args) {
        int x = 2;
        int n = 5;
        System.out.println(Arrays.toString(new Solution().solution(x, n)));
    }

    static class Solution {
        public long[] solution(int x, int n) {
            long[] answer = new long[n];
            long num = x;
            for(int i = 0; i < n; i++){
                answer[i] = num;
                num += x;
            }

            return answer;
        }
    }
}
